src/mom5/ocean_core/ocean_bbc.F90

```diff
@@ -68,6 +68,7 @@ module ocean_bbc_mod
 !  For determining bottom drag coefficient using a map of the roughness length.
 !  This approach is more relevant for coarse models
 !  than the constant roughness length used in the cdbot_law_of_wall option. 
+!  cdbot_lo <= cdbot(i,j) <= cdbot_hi.
 !  Default is cdbot_roughness_length=.false. 
```

Comment:
Should this be changed also?

Reply:
changed to what?

Reply:
Actually it was line 72 I thought I'd highlighted. To the same as there other changes I guess?

Reply:
Sorry, I'm still not getting what you're suggesting. 
What needs changing in line 72: 
`!  Default is cdbot_roughness_length=.false.`
?

Reply:
@aidanheerdegen not sure I understood what you were asking here, but are you happy for me to merge this?

Reply:
Actually I was mistaken. Sorry for causing confusion and delay (Thomas the Tank Engine style).

Sure, merge away I say.

Reply:
good-o - merged!